src/software/ai/passing/cost_function.cpp

```diff
@@ -13,7 +13,19 @@
 #include "software/geom/algorithms/convex_angle.h"
 #include "software/logger/logger.h"
 
-double ratePass(const World& world, const Pass& pass, const Rectangle& zone,
+double rateGroundPass(const World& world, const Pass& pass, const Rectangle& zone,
+                TbotsProto::PassingConfig passing_config)
+{
+    return ratePass(world, pass, zone, world.enemyTeam(), passing_config);
+}
+
+double rateChipPass(const World& world, const ChipPass& pass, const Rectangle& zone,
+                TbotsProto::PassingConfig passing_config)
+{
+    return ratePass(world, pass, zone, world.enemyTeam(), passing_config);
+}
```

Comment:
if you're using the same implementation, does it make sense to have separate `rateGroundPass` and `rateChipPass` methods?